crates/bevy_pbr/src/light_probe/environment_map.wgsl

```diff
@@ -1,16 +1,21 @@
 #define_import_path bevy_pbr::environment_map
 
+#import bevy_pbr::light_probe::query_light_probe
 #import bevy_pbr::mesh_view_bindings as bindings
 #import bevy_pbr::mesh_view_bindings::light_probes
 
 struct EnvironmentMapLight {
     diffuse: vec3<f32>,
     specular: vec3<f32>,
+    // Whether any applicable reflection probe was found.
+    found: bool,
```

Comment:
I don't think we need the found field. You can just initialize the diffuse/specular to 0 when nothing is found, and return early in other cases. Cleaner and a bit less branching.

Reply:
Good point, I didn't realize that you might as well ignore a light probe that provides zero light. I'll do that.